index/templates/index/index.html

```diff
@@ -33,6 +33,13 @@
 
 {% block body_block %}
   <!-- clock -->
+  <script>
+      if ('{{ alert_info }}' == 'mailbox'){
+          message = 'Please go to your mailbox and click the confirmation ' + 
+          'link;otherwise , your account would not be activated!';
+          alert(message);
+      }
+  </script>
```

Comment:
Prefer to make this into a function and call it in js file.
Let HTML code separates javascript code as possible.